### PR TITLE
feat(mobile-vitals): Add mobile vitals cards to performance, again.

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
This is a revert of a revert (redoing a previous PR) to add mobile vital cards to performance. There was a check missing for the kind of field being passed to the card details lookup, and apdex with thresholds (eg. `apdex(300)`) was likely causing it to come back as undefined. Updated the type and fixed it so that even if the field is undefined, the page at least won't break and we'll capture it instead. 

New code is inside the second commit.

### CODE REVIEW

**Comment by shruthilayaj** [comment]
Should we split up frontend and backend changes into different PRs?

File: src/sentry/search/events/fields.py [hunk 1]
```diff
@@ -1153,9 +1181,25 @@ def normalize(self, value, params):
             if value in {"measurements_value", "span_op_breakdowns_value"}:
                 return ["arrayJoin", [value]]
 
+        if value in {MEASUREMENTS_FRAMES_SLOW_RATE, MEASUREMENTS_FRAMES_FROZEN_RATE}:
+            field = FIELD_ALIASES[value]
+            return field.get_expression(params)
```
Comment (shruthilayaj) [inline src/sentry/search/events/fields.py:1186]: Is this required? Aren't we able to resolve the expression without this by detecting it's a field alias and adding it to the selected columns?

**Comment by k-fish** [comment]
@shruthilayaj these backend changes are additive and the UI is under feature flags, nothing under active use should have a cross dependency.

**Review by Zylphrex — APPROVED** [review]
Thanks @k-fish ❤️

**Pull request merged**